src/main/kotlin/dev/tricht/gamesense/model/ScrollingText.kt

```diff
@@ -1,15 +1,16 @@
 package dev.tricht.gamesense.model
 
+import dev.tricht.gamesense.Main
 import dev.tricht.gamesense.Tick
+import java.util.prefs.Preferences
+
+var preferences: Preferences = Preferences.userNodeForPackage(ScrollingText::class.java)
 
 data class ScrollingText(
-    var _text: String
+    var _text: String,
+    var MAX_DISPLAY_TEXT_LENGTH: Int = if (preferences.get("clockIcon", "true")!!.toBoolean()) 21 else 12
```

Comment:
You're checking here only for the `clockIcon` preference, but that could also be the `songIcon` right? Depending on what is currently scrolling. So the calling side should probably pass the max text length.

Reply:
Yeah, that's a typo - It worked in testing and I missed it because I have the clock icon disabled.  Will push a correction for that, thanks.

Reply:
Can you also swap the 21 and 12 around? It's not scrolling atm